packages/react/src/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -253,13 +310,19 @@ const ToggleSwitch = React.forwardRef<HTMLButtonElement, React.PropsWithChildren
         sx={sxProp}
         {...rest}
       >
-        {loading ? <Spinner size="small" /> : null}
+        <VisuallyHidden>
+          <AriaStatus announceOnShow id={loadingLabelId}>
+            {isLoadingLabelVisible && loadingLabel}
+          </AriaStatus>
+        </VisuallyHidden>
```

Comment:
Would the logic for message showing up be similar to:

```suggestion
        {isLoadingLabelVisisble ? <VisuallyHidden>
          <AriaStatus announceOnShow id={loadingLabelId}>
            {loadingLabel}
          </AriaStatus>
        </VisuallyHidden> : null}
```

Or is this not the same? Only reason this came up was with `AriaStatus` we wouldn't necessarily want the false-y case (loading label not visible) be enqueued to announce

Reply:
I guess the main difference is, with this change the live region isn't added to the DOM until after a user clicks the toggle and `isLoadingLabelVisisble` becomes `true`, compared to now where it's added during the initial render alongside the component.